cloud-native-app-developer/reading-list-front-end/src/api/books/instance.ts

```diff
@@ -14,9 +14,9 @@
 // specific language governing permissions and limitations
 // under the License.
 
-
-import { initInstance } from "../instance";
+import axios from "axios";
 
 export const getReadingListInstance = () => {
-  return initInstance(import.meta.env.VITE_CHOREO_BACKEND_URL);
+  let baseUrl = window.config.choreoApiUrl;
```

Comment:
Can't we directly use `window.config.choreoApiUrl` in line 21 without defining a variable, since this value is used only in 1 place.